### ISSUE TITLE
Creating new folders option is missing

### ISSUE THREAD

**Issue by iszhi** [eikek/docspell#2036]
It seems like that creating a new folder is not convinent while editing propertity for a new file. Can we add a similar function for Folder like we have for tags?
![image](https://user-images.githubusercontent.com/33443222/230907255-f52b324e-3300-474f-839f-4b740ae03f41.png)

**Comment by eikek** [comment]
Yes, that is missing!. It already exists as #1776

**Comment by iszhi** [comment]
I can feel the more effort. And, Actually I'm wondering if there is a way to export all Documents according to our assigned folders?

**Comment by eikek** [comment]
> there is a way to export all Documents according to our assigned folders?

Yes, that is possible using the client dsc. It has an export subcommand that can symlink into folders. Check out `dsc export --help`
